### PR TITLE
Cloud Tasks app

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

**Comment by averikitsch** [comment]
@kurtisvg or @dzlier-gcp do you mind reviewing this today?

File: appengine-java8/tasks/app/src/main/java/com/example/task/Enqueue.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    urlPatterns = "/taskqueues/enqueue")
+public class Enqueue extends HttpServlet {
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+      // TODO(developer): Replace these variables before running the sample.
+      String projectId = "my-project-id";
```
Comment (kurtisvg) [inline appengine-java8/tasks/app/src/main/java/com/example/task/Enqueue.java]: I think these should be `static final` and scoped for the class.

File: appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+public class Worker extends HttpServlet {
+
+  private static final Logger log = Logger.getLogger(Worker.class.getName());
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    // Do something with key.
+    log.info("Worker is processing " + key);
```
Comment (kurtisvg) [inline appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java]: I don't understand what this Servlet is supposed to be doing. Can you add a comment explaining what the goal of this endpoint is and how to plays into the larger app?

Comment (averikitsch) [reply appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java]: I updated the servlet description. Is it descriptive enough?

File: appengine-java8/tasks/app/src/main/java/com/example/task/Enqueue.java (hunk 1 above)
Comment (averikitsch) [reply appengine-java8/tasks/app/src/main/java/com/example/task/Enqueue.java]: done.

File: appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+@WebServlet(
+    name = "TaskWorker",
+    description = "Endpoint to process Cloud Task requests",
+    urlPatterns = "/taskqueues/worker"
+)
+public class Worker extends HttpServlet {
+
+  private static final Logger log = Logger.getLogger(Worker.class.getName());
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
```
Comment (kurtisvg) [inline appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java]:
```suggestion
  // The queued Task will trigger processing by sending this function a POST request. 
  protected void doPost(HttpServletRequest request, HttpServletResponse response)
```

File: appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java (hunk 2 above)
Comment (kurtisvg) [reply appengine-java8/tasks/app/src/main/java/com/example/task/Worker.java]: Made a suggestion. PTAL.

**Pull request merged**